src/contracts/core/StrategyManager.sol

```diff
@@ -515,6 +480,10 @@ contract StrategyManager is
     ) internal returns (bytes32) {
         require(strategies.length == shares.length, "StrategyManager.queueWithdrawal: input length mismatch");
         require(withdrawer != address(0), "StrategyManager.queueWithdrawal: cannot withdraw to zero address");
+        require(
```

Comment:
can we remove the `receiveAsTokens` argument now? avoids complexity that will never be used now

Reply:
refering to completeWithdrawal ofc

Reply:
I think we still want something like this, to deal with the edge case of a reverting strategy -- allowing users to "cancel" a withdrawal still seems good, which is essentially what marking `receiveAsTokens = false` is.